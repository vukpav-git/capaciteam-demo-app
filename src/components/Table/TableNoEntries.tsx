import { Typography } from "@mui/material";

const TableNoEntries = () => (
  <Typography
    marginLeft="10px"
    marginTop="4px"
    fontSize={16}
    fontFamily="monospace"
  >
    No entries found.
  </Typography>
);

export default TableNoEntries;
