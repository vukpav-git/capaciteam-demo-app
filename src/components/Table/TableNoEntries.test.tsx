import { render, screen } from "@testing-library/react";

import TableNoEntries from "./TableNoEntries";

describe("Table No Entries tests", () => {
  it("should match snapshot", () => {
    const { asFragment } = render(<TableNoEntries />);

    expect(asFragment()).toMatchSnapshot();
  });

  it('renders the component with the text "No entries found!"', () => {
    render(<TableNoEntries />);
    const messageElement = screen.getByText(/No entries found./i);
    expect(messageElement).toBeInTheDocument();
    expect(messageElement).toHaveStyle({
      marginLeft: "10px",
      marginTop: "4px",
      fontSize: "16px",
      fontFamily: "monospace",
    });
  });
});
