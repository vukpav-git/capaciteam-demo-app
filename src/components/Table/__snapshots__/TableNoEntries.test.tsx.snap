// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table No Entries tests should match snapshot 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body1 css-mrmrsa-MuiTypography-root"
  >
    No entries found.
  </p>
</DocumentFragment>
`;
